### PR TITLE
Added Android manual linking guide

### PR DESCRIPTION
**Pull request by rommyarb**
Just doing my part

### CODE REVIEW

File: docs/Linking.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
 # Manual Linking
+-
```
Comment (Krizzu) [inline docs/Linking.md]: We don't need that here

```suggestion

```

File: docs/Linking.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
 
 ## iOS
+-
```
Comment (Krizzu) [inline docs/Linking.md]: Same as above
```suggestion

```

File: docs/Linking.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
...
 
-## Android
\ No newline at end of file
+## Android
+1. Edit `android/settings.gradle` to look like this (without the +):
```
Comment (Krizzu) [inline docs/Linking.md]: We don't have to be that explicit :)

```suggestion
1. Add project to `android/settings.gradle`:
```

File: docs/Linking.md [hunk 4]
```diff
@@ -1,5 +1,41 @@
...
+```diff
+rootProject.name = 'MyApp'
+
+include ':app'
+
++ include ':@react-native-community_async-storage'
++ project(':@react-native-community_async-storage').projectDir = new File(rootProject.projectDir, '../node_modules/@react-native-community/async-storage/android')
+  ```
+
+2. And in `android/app/build.gradle` add this to dependencies:
```
Comment (Krizzu) [inline docs/Linking.md]: I might be picky here, but let's don't start a sentence with conjunction :)
```suggestion
2. In `android/app/build.gradle` add to dependencies:
```

**Comment by rommyarb** [comment]
> Great work!
> 
> I've left a few suggestions to consider, let me know what you think.
> Also, please follow PR template on your next contribution :).
> 
> Thanks.

Thank you for your guidance. I'm sorry I didn't follow the PR template. I'm new to git world. I hope I get the lesson. Thank you for this opportunity!

**Pull request merged**